### PR TITLE
SAK-29000 Create sakai.property to show/hide authorization fields when adding roster(s) to a course site

### PR DESCRIPTION
**Pull request by bjones86**
https://jira.sakaiproject.org/browse/SAK-29000

Site Info -> Edit Rosters -> Add Roster

The authorization fields are not actually required and can create roster attachment issues (waiting for authorization).

This introduces a sakai.property (site.addroster.authorizationrequired) to control whether or not these fields are visible to the user. Default is true (original functionality). Only when set to false are the fields not visible to the user.

### CODE REVIEW

File: site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java [hunk 1]
```diff
@@ -3215,6 +3218,10 @@ public int compare(String s1, String s2) {
 			context.put("termCourseList", state
 					.getAttribute(STATE_TERM_COURSE_LIST));
 
+			// mweston4 - OWL-657
```
Comment (ottenhoff) [inline site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java]: your local institutional JIRAs shouldn't be part of this commit ;)

File: config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties [hunk 2]
```diff
@@ -2660,6 +2660,11 @@
 # DEFAULT: 10
 #site.adminperms.sitesuntilpause=10
 
+### SAK-29000
+# Determines if the authorization fields are presented to the user in Site Info -> Edit Rosters -> Add Roster
+# DEFAULT: true
+# site.addroster.authorizationrequired=true
```
Comment (ottenhoff) [inline config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties]: Comment pattern in this document has been to set the commented out property to the opposite of the default.

File: site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java (hunk 1 above)
Comment (bjones86) [inline site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java]: Haha, yikes! Good catch, thanks Sam.

File: config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties (hunk 2 above)
Comment (bjones86) [inline config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties]: I'll fix this one up too. Cheers

**Comment by bjones86** [comment]
Updated as per Sam's comments
